packages/shared/src/components/post/PostHeaderActions.tsx

```diff
@@ -130,7 +123,7 @@ export function PostHeaderActions({
         <Button
           className={classNames('btn-tertiary', !inlineActions && 'ml-auto')}
           icon={<MenuIcon />}
-          onClick={(event) => showPostOptionsContext(event)}
+          onClick={(event) => onMenuClick(event)}
```

Comment:
I think we can simplify this with the suggestion below.
```suggestion
          onClick={onMenuClick}
```